src/gui/mzroll/mainwindow.cpp

```diff
@@ -1138,6 +1141,18 @@ void MainWindow::setQComboBox() {
 	else ionizationModeComboBox->setCurrentIndex(2);
 }
 
+void MainWindow::setInjectionOrderFromTimeStamp() {
+
+    vector<mzSample*> samples = getSamples();
+    std::sort(samples.begin(), samples.end(), mzSample::compInjectionTime);
```

Comment:
is there a reason for using  a custom function for comparing? std::sort by default sorts in ascending order

Reply:
As it is a user-defined class, we can't use default sorts here.

Reply:
oh correct. my bad

Reply:
In future we should use lambdas for such things . let's try not to use static functions unless they are really required

Reply:
@rish9511 Yes, makes sense.